src/Form/matching-configuration.js

```diff
@@ -257,7 +257,7 @@ const matchingConfiguration = {
             matchers: {
                 email: {match: '.mail\\b', skip: 'phone|name|reservation number', forceUnknown: 'search|filter|subject|title|\btab\b'},
                 password: {match: 'password', forceUnknown: 'captcha|mfa|2fa|two factor'},
-                username: {match: '(user|account|apple|login)((.)?(name|id|login).?)?(.?(or|/).+)?$|benutzername', forceUnknown: 'search|policy'},
+                username: {match: '(user|account|apple|login|net)((.)?(name|id|login).?)?(.?(or|/).+)?$|benutzername', forceUnknown: 'search|policy'},
```

Comment:
Apparently, `NetID` is a common way to refer to usernames for several academic institutions in the US. See [this](https://app.asana.com/0/1199178362774117/1203313902232286/f) for more info.